staging/src/k8s.io/apimachinery/pkg/util/sets/set.go

```diff
@@ -177,61 +172,28 @@ func (s1 Set[T]) Equal(s2 Set[T]) bool {
 	return len(s1) == len(s2) && s1.IsSuperset(s2)
 }
 
-type sortableSlice[T SetTypes] struct {
-	slice []T
-	OrderedFuncs[T]
-}
-
-func (s sortableSlice[T]) Len() int {
-	if s.LenFunc == nil {
-		return len(s.slice)
-	}
-	return s.LenFunc(s.slice)
-}
-func (s sortableSlice[T]) Less(i, j int) bool {
-	if s.LessFunc == nil {
-		// unsorted if no less func specified
-		return true
-	}
-	return s.LessFunc(s.slice, i, j)
-}
-func (s sortableSlice[T]) Swap(i, j int) {
-	if s.SwapFunc == nil {
-		s.slice[i], s.slice[j] = s.slice[j], s.slice[i]
-		return
+// List returns the slice with contents in random order.
+func (s Set[T]) List() []T {
```

Comment:
I recommend omitting this function completely, it is a land mine. The name needs to have the word "Unsorted" in it.